src/components/StoreInfo/StoreMerchTab.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import { BrowsePageSeller } from '../../utilities';
+
+export interface Props {
+  getStoreImages: Function;
+  seller: BrowsePageSeller;
+}
+
+const StoreMerch: React.SFC<Props> = ({ seller, getStoreImages }) => {
+  //   const images = getStoreImages(seller.seller_id);
+
+  return (
+    <MerchContainer>
+      <p>
+        Support {seller.name} by purchasing custom clothing and stickers
```

Comment:
Since this page isn't shown yet, no need to fix this now. But could you put a TODO to translate this page?